tests/testthat/test-regressions.R

```diff
@@ -259,7 +259,7 @@ test_that("calls to make_afun within loop work correctly", {
 })
 
 
-test_that("keeping non-existent levels doesn't break internal machinery", {
+test_that("keeping non-existent levels doesn't break internal machinery but errors", {
```

Comment:
This regression test was added for #136, but allowing the selection of levels not present in the data is a potential issue for the final table. Corrected with the new handling.